app/config/environments.php

```diff
@@ -83,12 +87,19 @@
         'build' => '/usr/src/code/docker/environments/deno-1.6',
         'logo' => 'deno.png',
     ],
-    // 'dart-2.8' => [
-    //     'name' => 'Dart',
-    //     'version' => '2.8',
-    //     'base' => 'google/dart:2.8',
-    //     'image' => 'appwrite/env-dart:2.8',
-    //     'build' => '/usr/src/code/docker/environments/dart-2.8',
-    //     'logo' => 'dart.png',
-    // ],
-];
\ No newline at end of file
+    'dart-2.10.4' => [
+        'name' => 'Dart',
+        'version' => '2.10.4',
```

Comment:
It will be better to use minor version (2.10) and not full version (2.10.4), to avoid too many supported envs.